framework/src/streaming/lib/msk/msk-utils.ts

```diff
@@ -126,6 +126,16 @@ export class KafkaVersion {
    */
   public static readonly V3_6_0 = KafkaVersion.of('3.6.0');
 
+  /**
+   * Kafka version 3.7.x
+   */
+  public static readonly V3_7_X = KafkaVersion.of('3.7.x');
```

Comment:
Why `.x`, shouldn't we have each each one indepently, like `3.7.1` or `3.7.2` in the future?

Reply:
this are the allowed value by the aws-cli 
`2.8.2.tiered, 2.4.1.1, 3.3.2, 3.5.1, 3.6.0, 3.1.1, 3.2.0, 3.3.1, 3.4.0, 2.1.0, 3.7.x, 2.6.2, 2.7.1, 2.8.0, 2.6.3, 2.7.2, 2.8.1, 2.5.1, 2.6.0, 2.6.1, 2.7.0, 2.3.1, 2.4.1, 3.7.x.kraft, 2.2.1, 1.1.1`